src/dashboards/videos/course.jsonnet

```diff
@@ -0,0 +1,317 @@
+      bucketAggs=[common.object.date_histogram('$STATEMENTS_INTERVAL')],
+      timeField='timestamp'
+    )
+  ),
+  gridPos={ x: 12, y: 9, w: 12, h: 9 }
+)
+.addPanel(
+  {
+    title: 'Statements by user',
+    description: |||
+      The count of statements by user.
+      On the X-axis is the number of statements.
+      On the Y-axis is the number of users.
+    |||,
+    datasource: common.lrs,
+    fieldConfig: {
+      defaults: {
+        color: {
+          mode: 'palette-classic',
+        },
+        displayName: 'User',
+        mappings: [],
+      },
+      overrides: [],
+    },
+    options: {
+      bucketOffset: 0,
+      legend: {
+        calcs: ['count', 'max', 'mean'],
+        displayMode: 'list',
+        placement: 'bottom',
+      },
+    },
+    pluginVersion: '8.0.3',
+    targets: [
+      {
+        alias: '',
+        bucketAggs: [
+          {
+            field: 'timestamp',
+            id: '2',
+            settings: {
+              interval: '3600000',
+              min_doc_count: '1',
+            },
+            type: 'histogram',
+          },
+          {
+            field: common.field.actor_account_name,
+            id: '3',
+            settings: {
+              min_doc_count: '1',
+              order: 'desc',
+              orderBy: '_term',
+              size: '0',
+            },
+            type: 'terms',
+          },
+        ],
+        metrics: [common.object.count_metric],
+        query: common.query.school_course_session,
+        refId: 'A',
+        timeField: 'timestamp',
+      },
+    ],
+    transformations: [
+      {
+        id: 'groupBy',
+        options: {
+          fields: {
+            Count: {
+              aggregations: [
+                'sum',
+              ],
+              operation: 'aggregate',
+            },
+            [common.field.actor_account_name]: {
+              aggregations: [],
+              operation: 'groupby',
+            },
+          },
+        },
+      },
+    ],
+    type: 'histogram',
+  },
+  gridPos={ x: 12, y: 18, w: 12, h: 9 }
+)
+.addPanel(
+  {
+    title: 'Completed videos by user',
+    description: |||
+      The distribution of the number of times users completed videos.
+      On the X-axis is the number of completed videos.
+      On the Y-axis is the number of users.
+    |||,
+    datasource: common.lrs,
+    fieldConfig: {
+      defaults: {
+        color: {
+          mode: 'palette-classic',
+        },
+        displayName: 'User',
+        mappings: [],
+      },
+      overrides: [],
+    },
+    options: {
+      bucketOffset: 0,
+      legend: {
+        calcs: ['count', 'max', 'mean'],
+        displayMode: 'list',
+        placement: 'bottom',
+      },
+    },
+    pluginVersion: '8.0.3',
+    targets: [
+      {
+        alias: '',
+        bucketAggs: [
+          {
+            field: 'timestamp',
+            id: '2',
+            settings: {
+              interval: '3600000',
```

Comment:
Is ms the unit? This stands for 1h?

Reply:
Nice catch) Thanks) Indeed we can write `1h` instead of `3600000`) Updated)

Reply:
Apropos, I made a mistake. it seems grafana don't support writing `1h`  instead of  `3600000` (I got parsing errors when visiting the dashboard)